src/Nethermind/Nethermind.Optimism.Test/Rpc/OptimismEthRpcModuleTest.cs

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Threading.Tasks;
+using Nethermind.Blockchain.Synchronization;
+using Nethermind.Config;
+using Nethermind.Core;
+using Nethermind.Core.Extensions;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Crypto;
+using Nethermind.Facade;
+using Nethermind.Facade.Eth;
+using Nethermind.JsonRpc.Client;
+using Nethermind.JsonRpc.Modules.Eth.FeeHistory;
+using Nethermind.JsonRpc.Test.Modules;
+using Nethermind.Logging;
+using Nethermind.Optimism.Rpc;
+using Nethermind.Serialization.Rlp;
+using Nethermind.Synchronization.ParallelSync;
+using Nethermind.TxPool;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Nethermind.Optimism.Test.Rpc;
+
+public class OptimismEthRpcModuleTest
+{
+    [Test]
+    public async Task Sequencer_send_transaction_with_signature_will_not_try_to_sign()
```

Comment:
This test is pretty much a copy/paste of:

https://github.com/NethermindEth/nethermind/blob/2d9dd7c7fcf5f8f2d5b36eeeccb85ef8862fb52f/src/Nethermind/Nethermind.JsonRpc.Test/Modules/Eth/EthRpcModuleTests.cs#L1030